apps/etterlatte-behandling/src/main/kotlin/behandling/sjekkliste/Sjekkliste.kt

```diff
@@ -62,5 +62,27 @@ internal val defaultSjekklisteItemsOMS =
         "Bosatt utland: Opprettet oppgave til NØP om kildeskatt",
     )
 
-internal val defaultFoerstegangsbehandlingItemsOms =
-    defaultSjekklisteItemsOMS + "Sjekk om bruker har overgangsstønad"
```

Comment:
Er det meningen at "Sjekk om bruker har overgangsstønad" fjernes fra førstegangsbehandling?

Reply:
Nei, sjekkliste punktene for førstegangsbehandlingene skal ikke endres. Kan hende at jeg har vært litt for rask i refactoringen